README.md

```diff
@@ -219,8 +219,8 @@ the same time :-). I want to add an edited and higher-quality video, too.
 
 1. Place your Teensy 3.x or 4.x on top of the pin header and solder all its pins.
 
-   * If you’re using a Teensy++ 2.0, you must not connect some pins! [See the
-     instructions below](#soldering-instructions-for-the-teensy-20).
+   * If you’re using a Teensy++ 2.0, you may want to remove some pins before
```

Comment:
Here and below, I think we can make this language stronger: there’s no need to keep the plier method around. Let’s just recommend people close JP1-JP3 and open JP4-JP6 for Teensy 2.0 support.